### PR TITLE
Update Swagger configuration to publish correct APIs

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1413

### Change Description ###
Expose only `sas token` and `upload supplier statement (reconciliation)` endpoints in swagger spec.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/api-mgmt.tf [hunk 1]
```diff
@@ -36,7 +36,7 @@ module "api_mgmt" {
     "https"
   ]
   service_url = "http://${var.product}-${var.component}-${var.env}.service.core-compute-${var.env}.internal"
-  swagger_url = "https://hmcts.github.io/reform-api-docs/specs/blob-router-service.json"
+  swagger_url = "https://hmcts.github.io/reform-api-docs/specs/blob-router-service.json?v2"
```
Comment (aliveni) [inline infrastructure/api-mgmt.tf:39]: to refresh the APIs in API management

File: src/main/java/uk/gov/hmcts/reform/blobrouter/config/SwaggerConfiguration.java [hunk 2]
```diff
@@ -22,7 +22,8 @@ public Docket api() {
             .select()
             .apis(RequestHandlerSelectors.basePackage(BASE_PACKAGE + ".controllers")
                       .or(RequestHandlerSelectors.basePackage(BASE_PACKAGE + ".reconciliation.controller")))
-            .paths(PathSelectors.any())
+            .paths(PathSelectors.ant("/token/*")
+                       .or((PathSelectors.ant("/reconciliation-report/*"))))
```
Comment (aliveni) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/config/SwaggerConfiguration.java:26]: image

File: src/main/java/uk/gov/hmcts/reform/blobrouter/config/SwaggerConfiguration.java [hunk 3]
```diff
@@ -22,7 +22,8 @@ public Docket api() {
...
             .build();
```
Comment (aliveni) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/config/SwaggerConfiguration.java:27]: https://reform-scan-blob-router-pr-729.service.core-compute-preview.internal/v2/api-docs

File: infrastructure/api-mgmt.tf (hunk 1 above)
Comment (mdayican) [reply infrastructure/api-mgmt.tf:39]: will revert?

**Review by mdayican — APPROVED** [review]

**Pull request merged**